doc/Translations.md

```diff
@@ -0,0 +1,49 @@
+# Translating Warzone 2100
+
+- [How do I help translate?](#how-do-i-help-translate)
+- [Notes for translators](#notes-for-translators)
+- [Notes for proofreaders](#notes-for-proofreaders)
+- [Notes for developers](#notes-for-developers)
+- [For developers merging translation PRs](#for-developers-merging-translation-prs)
+
+## How do I help translate?
+Help us get Warzone 2100 translated into your native language and reach more people.
+- Browse over to our Crowdin project: https://crowdin.com/project/warzone2100
+- Select the language you’d like to contribute to / review.
+   - If the language you want isn’t listed, please open a new issue at: https://github.com/Warzone2100/warzone2100/issues/new
+- If you don’t have a Crowdin account yet, you can sign up with an email, or just log-in using another account (GitHub, Google, Twitter, etc).
+- Request access to the languages you’d like to help contribute translations to!
+
+## Notes for translators:
+- All translation happens in the Crowdin project. Do not open pull requests directly on the warzone2100 repository for translation updates.
```

Comment:
Why do you insist that everybody must use Crowdin although nobody does?
Editing PO files and uploading them is more convenient. If only a single
person is translating (as is usually the case), there is no need for a
translation platform. Teams of multiple translators can use GitHub.

The most popular way for new translators to submit their work is to post
in the forum thread ["Translation: Announcements / coordination"](http://forums.wz2100.net/viewtopic.php?f=6&t=4418).
Having assisted several translators in recent years, my observation has
been that the main difficulty for them is getting started. A lower entry
barrier might be achieved with new tutorials, more prominently displayed
translation threads in our forums and additional information shown in
main menu screens. But forcing people to sign up on an unknown platform
before they can begin to contribute will discourage them from doing so.

Reply:
> Why do you insist that everybody must use Crowdin although nobody does?
Editing PO files and uploading them is more convenient. If only a single
person is translating (as is usually the case), there is no need for a
translation platform. Teams of multiple translators can use GitHub.

(Please keep in mind that this document is a draft, and is intended for new folks who might want to contribute.)

A big goal is trying to automate as much of everything as possible (and reducing maintenance / upkeep burden on the very limited time of currently-active volunteers). Crowdin provides a lot of helpful sanity checks (and makes it very easy to catch and fix errors in the translation strings). If we want to continue to speed up release cadence, and improvements, without things breaking left and right, it has some very useful tools.

While you may be comfortable editing .po files directly, many people aren't (and we still see broken .po files submitted, or issues with export settings in po editors, like maximum line length). Comparing changes / merging .po files can be annoying (certainly with a simple diff, depending on changes to source file / translation string ordering). Additionally, conversations / discussion over individual string translations (and possible alternatives / phrasing / etc) could benefit from a setup built for more collaboration. (And with more tools to catch mistakes.)

Finally, viewing translation history at the moment requires delving into Git and looking through all previous versions of a .po file - Crowdin makes it much easier to review how strings / translations have evolved.

I believe you can upload/download .po files from Crowdin - so if the ultimate decision were "we'd like to have everything moving through Crowdin to help keep track of changes prior to upstreaming to the GitHub repo" (which is what I'd like to test), I believe you'd be able to simply replace the final "create a PR on the main repo" step in your workflow with an "upload the *.po file to Crowdin" step.

But there's no better way to test all of this than to actually test it, and that's the step we're going towards.

(If you _want_ to still submit a .po file to this repo, that workflow works too - Crowdin will pull the changes from the .po file - so if there are people who want to just use Crowdin, everyone can work together. But I'd like to test having everything move through Crowdin for now, and tweak settings & expectations based on feedback / as necessary.)

EDIT: Also, Crowdin lets us easily list multiple (not just .po file) translation sources. As an example, the Windows installer strings will be included in Crowdin. (No need for anyone to figure out another file format.)

> Having assisted several translators in recent years, my observation has
> been that the main difficulty for them is getting started.

I agree! That's part of why we're testing this. There are a lot of other open source / commercial projects using Crowdin (or other translation platforms) that have seen a massive boost in contributions when they provide a simple, web-based way to review and contribute translations.

(In fact, a lot of the original translations for Warzone were done via [Launchpad](https://translations.launchpad.net). See: `X-Launchpad-Export-Date` in the .po files - you might have to delve into the Git history for some.)

> But forcing people to sign up on an unknown platform before they can begin to contribute will discourage them from doing so.

Part of why we're testing Crowdin is you don't need to sign up for an additional account - you can also just sign in with a Google, Twitter, GitHub (etc) account.